src/DynamoWPFCLI/Program.cs

```diff
@@ -16,6 +16,11 @@ static internal void Main(string[] args)
             {
                 var cmdLineArgs = StartupUtils.CommandLineArguments.Parse(args);
                 var locale = StartupUtils.SetLocale(cmdLineArgs);
+                if (cmdLineArgs.DisableAnalytics)
+                {
+                    Dynamo.Logging.Analytics.DisableAnalytics = true;
```

Comment:
`Dynamo.Logging.Analytics.DisableAnalytics = cmdLineArgs.DisableAnalytics;`?

Reply:
kind of the same thing
We only set it if `cmdLineArgs.DisableAnalytics` is true